### DISCUSSION TITLE
Random as a playback option or probability on kits

### DISCUSSION THREAD

**Discussion by Ponchomouse** [SynthstromAudible/DelugeFirmware#1223] [Ideas]
One great feature of the circuit tracks is that random is one of the playback options so you can quickly make a random drum fill. It would be great to get something along this lines so you set the probability of each sound. I notice if you hold down 5 notes on the grid and set probability to 20% it plays only 1 each time rather than all havign 20% chance of playing, maybe we could flesh out this feature.

**Comment by m-m-adams** [comment]
Hey you can do it by holding the row audition pad and setting probability - that will do it individually per drum. Holding multiple pads and setting probability makes them dependent on each other

**Comment by Ponchomouse** [comment]
Great thanks, not quite the same but works to get a similar result. And extension of this would be like the per drum random on the arturia kit like the drumbrute. It essentially let's you dial in a bit more randomness to the samples. So high hight might have an extra occasionally or miss one. Not sure how easy this it to program but really handy to keep things interesting. Also can imagine it working great on synth lines.
